compose/foundation/foundation/src/skikoMain/kotlin/androidx/compose/foundation/Scrollbar.skiko.kt

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Please note that such rename breaks binary compatibility for all file scope public functions

Reply:
Not fixed in this file